doc/common/chapter00.md

```diff
@@ -112,6 +112,19 @@ Any specification work created within CNTT **must** conform to the following pri
    - This pertains to features that operate at detailed levels of hardware granularity, such as EPA.
 
 <a name="2.2"></a>
+## Requirements Principles
+The agreed upon rules and recommendations to which a compliant NF or cloud infrastructure must adhere. 
+
```

Comment:
@xmulligan , Can we replace NF with workload

Reply:
thanks, updated